lib/tasks/route_import.rake

```diff
@@ -6,52 +6,29 @@ require 'csv'
 
 namespace :routes do
   task :import, [:csv_file] => :environment do |_, args|
-    Route.delete_all
-    BusStopsRoute.delete_all
-    route_hash = {}
+    stop_hash = {}
     CSV.foreach(args[:csv_file], headers: true, col_sep: ';') do |row|
-      stop = BusStop.find_by hastus_id: row['stp_identifier']
-      if stop.present?
-        route = Route.find_or_create_by number: row['rte_identifier'].strip
-
-        direction = row['direction']
-        variant = row['variant']
-        stop_id = row['stp_identifier']
-        sequence = row['stop_variant_rank']
-
-        route_hash[route] ||= {}
-        route_hash[route][direction] ||= {}
-        route_hash[route][direction][variant] ||= []
-        route_hash[route][direction][variant] << { stop_id => sequence }
-      end
+      route = row['rte_identifier']
+      stop = row['stp_identifier']
+      dir = row['direction']
+      variant = row['variant']
+      sequence = row['stop_variant_rank'].to_i
```

Comment:
Nitpick, but I might define the variables in the order you use them (or perhaps in the order they occur in the file).